frontend/src/components/ShootWorkers/MachineImage.vue

```diff
@@ -36,6 +36,9 @@ SPDX-License-Identifier: Apache-2.0
          {{item.name}} [{{item.version}}]
         </span>
       </template>
+      <template v-slot:message="{ message }">
+        <div v-html="message"></div>
```

Comment:
Ich würde das irgendwie so machen.
```suggestion
       <machine-image-hints v-if="isJSON(message)" :hints="JSON.parse(message)"/>
       <div v-text="message"/>
```

Reply:
CodeSandbox example:
https://codesandbox.io/s/focused-cdn-qn1wl?file=/src/components/MultiMessage.vue
```js
<template>
  <div class="wrapper">
    <template v-for="({ type, hint }, index) in hints">
      <div v-if="type === 'html'" v-html="hint" :key="index" />
      <div v-else v-text="hint" :key="index" />
    </template>
  </div>
</template>

<script>
export default {
  props: {
    message: {
      type: String,
    },
  },
  computed: {
    hints() {
      try {
        const obj = JSON.parse(this.message);
        console.log(obj);
        return Array.isArray(obj) ? obj : [obj];
      } catch (err) {
        return [
          {
            type: "text",
            hint: this.message,
          },
        ];
      }
    },
  },
};
</script>
```

Reply:
done